tools/deploy_partition.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # Lint as: python3
+"""Tool that is part of firmware upgrabability in OpenSK.
+"""
```

Comment:
Same as before, shoudln't this be all one one line?

Reply:
Fixed.